src/lib/doctrine-dbal-bulk/composer.json

```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
-        "doctrine/dbal": "^3.6"
+        "doctrine/dbal": "^3.8 || ^4.0"
```

Comment:
how much of an effort it would be to keep ^3.6?

Reply:
Could be not too big, but that version is not maintained anymore.